frontend/apps/filemanager/filemanagerfilesearcher.lua

```diff
@@ -165,8 +166,8 @@ function FileSearcher:showSearchResults()
         dimen = Screen:getSize(),
     }
     self.search_menu = Menu:new{
-        width = Screen:getWidth()-15,
-        height = Screen:getHeight()-15,
+        width = Screen:getWidth() - (Size.border.window * 4),
+        height = Screen:getHeight() - (Size.border.window * 4),
```

Comment:
This `Size.border.window * 4` should probably be a new `Size.padding.fullscreen_popout * 2`

Reply:
Or rather Size.margin.fullscreen_popout (with CSS, margin is outside the borders, padding is inside :)